test/arcade/potts/env/location/PottsLocationTest.java

```diff
@@ -1294,31 +1337,142 @@ public void separateVoxels_validLists_updatesCenter() {
         assertEquals(1. / 3, split.cy, EPSILON);
         assertEquals(3. / 3, split.cz, EPSILON);
     }
-    
+
     @Test
-    public void split_noOffsetsNoDirection_callsMethods() {
-        PottsLocation spy = spy(new PottsLocationMock(voxelListAB));
-        spy.split(RANDOM);
-        verify(spy).split(eq(RANDOM), eq(null), eq(null), eq(0.5));
+    public void split_noOffsetsNoDirectionRandomDoubleZero_splitsVoxelsCorrectly() {
```

Comment:
```suggestion
    public void split_noOffsetsNoDirection_splitsVoxelsCorrectly() {
```

For consistency with analogous tests. You are not also checking `split_noOffsetsNoDirectionRandomDoubleOne_splitsVoxelsCorrectly` so no need for it to be in the name